roles/openshift_hosted/tasks/registry/registry.yml

```diff
@@ -40,6 +40,9 @@
     openshift_hosted_registry_images: "{{ openshift.hosted.registry.registryurl | default('openshift3/ose-${component}:${version}')}}"
     openshift_hosted_registry_volumes: []
     openshift_hosted_registry_env_vars: {}
+    openshift_hosted_registry_route_certificates: "{{ openshift.hosted.registry.route.certificates | default(false) }}"
+    openshift_hosted_registry_route_host: "{{ openshift.hosted.registry.route.host | default(false) }}"
```

Comment:
I guess I'm just not sure why we'd expose two different routes...  Wouldn't we just want to expose one and have it use the type of route encryption settings we request?